sources/platform/actors/development/deployment/source_types.md

```diff
@@ -5,69 +5,92 @@ slug: /actors/development/deployment/source-types
 
-If the Actor's source code is hosted externally in a Git repository, it can consist of multiple files and directories, use its own **Dockerfile** to control the build process (see [Custom Dockerfile](../actor_definition/docker.md#custom-dockerfile) for details) and have a user description in store fetched from the <strong>README.md</strong> file. The location of the repository is specified by the **Git URL** setting, which can be an **https**, **git** or **ssh** URL.
+Hosting your Actor's source code in a Git repository allows for multiple files and directories, a custom `Dockerfile` for build process control, and a user description fetched from `README.md`. Specify the repository location using the **Git URL** setting with `https`, `git`, or `ssh` protocols.
+
+To deploy an Actor from GitHub, set the **Source Type** to **Git repository** and enter the GitHub repository URL in the **Git URL** field. You can optionally specify a branch or tag by adding a URL fragmend (e.g., `#develop`).
+
+To use a specific directory, add it after the branch/tag, separated by a colon (e.g., `#develop:some/dir`)
+
+:::note GitHub integration
+
+You can easily set up an integration where the Actor is automatically rebuilt on every commit to the Git repository. For more details, see [GitHub integration](platform/integrations/github).
+
+:::
+
+### Private repositories
+
+When using a private Git repository for your Actor's source code, you need to configure a deployment key. This key grants Apify secure, read-only access to your repository.
+
+:::note What are deployment keys
+
+Deployment keys are unique SSH keys that allow Apify to clone and build your Actor from a private repository.
+
+:::
 
-[//]: # (TODO: It's pretty outdated, we should probably update the Actor too)
-To help you get started quickly, you can use the [apify/quick-start](https://apify.com/apify/quick-start) Actor which contains all the boilerplate necessary when creating a new Actor hosted on Git. The source code is available on [GitHub](https://github.com/apify/actor-quick-start).
+#### How to configure deployment keys
 
-To specify a Git branch or tag to check out, add a URL fragment to the URL. For example, to check out the **develop** branch, specify a URL such as `https://github.com/jancurn/some-actor.git#develop`.
+To configure the deployment key for your Actor's source code you need to:
 
-Optionally, the second part of the fragment in the Git URL (separated by a colon) specifies the directory from which the Actor will be built (and where the `.actor`) folder is located. For example, `https://github.com/jancurn/some-actor.git#develop:some/dir` will check out the **develop** branch and set **some/dir** as the root directory of the Actor.
+1. In your Actor's settings, find the **Git URL** input field
+1. Click the **deployment key** link below the input field
+1. Follow the instruction to add the key to your Git hosting service.
```

Comment:
```suggestion
3. Follow the instruction to add the key to your Git hosting service.
```

Reply:
Docusaurus would change those into proper numbers, but I guess we can have it this way also. Will fix, thank you!